common/workunit/workunit.cpp

```diff
@@ -6894,11 +6894,17 @@ bool extractFromWorkunitDAToken(const char * distributedAccessToken, StringBuffe
 //   1 : Signature does not verify (wuid/username don't match, or signature does not verify)
 //   2 : Workunit not active
 //   Throws if unable to open workunit
-wuTokenStates verifyWorkunitDAToken(const char * distributedAccessToken)
+wuTokenStates verifyWorkunitDAToken(const char * _user, const char * _distributedAccessToken)
 {
+    if (isEmptyString(_user) || isEmptyString(_distributedAccessToken))
```

Comment:
What happens on a system that doesn't have security enabled?  This will probably cause additional failures.
If it is blank in the access token, and the user is blank then that should match.

E.g.
                 distributedAccessToken="HPCC[u=,w=W20190221-190418];"

 from my local machine.

Reply:
Good catch, you are correct